### DISCUSSION TITLE
We have to delete MyTourbook installation files

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1322] [General]
Today I received the below request from Sourceforge, do you have any suggestions which files should be deleted forever?

"Hello,

We are contacting you because you are listed as an Admin for https://sourceforge.net/projects/mytourbook/

This project is using a large amount of storage.

We need your help removing a lot of your old releases.
Please remove as much as you can of your old releases and unused files within 14 days.

You can use this tool to help you identify the files that aren't getting many downloads and take up a lot of space.
https://sourceforge.net/p/mytourbook/admin/files/large_files

We are letting you know so you can choose which files to remove.
Please be aware that if we don't see any difference after 14 days, we will have to remove all the files and folders with 0 or a few monthly downloads ourselves.

Thank you for your cooperation

Sincerely,

SourceForge Support"

This is from the admin site:

![image](https://github.com/mytourbook/mytourbook/assets/1283445/6674ebd1-37b0-44f2-8051-8c08642b5f96)

**Comment by mff73** [comment]
Hi Wolfgang,
My only personnal thinking : it has effectively not so much sense to let full history available to the wild public : who will download and use any very old build ?
But then, how to choose what to delete, if any old build could be usefull for someone ?
One idea (i am not any expert, just out pure logic) : keep what could not be rebuilt easily, and remove those from 1,2 or 3 years old which could be rebuilt eventually "on specific" demand (just in case any prehistoric user would really need it :) ) ?

**Comment by wolfgang-ch** [comment]
These are my thoughts, delete all except

* Keep all for the last n years, with n e.g. 5
* Keep one release for each year

**Comment by stvzldn** [comment]
I believe some releases may be specific to particular OS and possibly those be retained (e.g. MAC or Linux) (I am using only Windows) or a specific version if Java.
Otherwise if the tour database can always be updated to the most recent then the last n years sound good.

Are these only installable release versions or full project buildable code?

**Reply by wolfgang-ch** [reply]
> Are these only installable release versions or full project buildable code?

These are only instllable releases. The project buildable code is in this Github repo and has a tag for each release.

**Comment by FJBDev** [comment]
My suggestion would be to stop using sourceforge and post the releases on
Github.

I like to use old releases when i investigate when a bug appeared but its
rare.

Frederic

**Comment by wolfgang-ch** [comment]
Currently I'm uploading MT release files from the last few years to github and will delete them from sourceforge. 

You may now get notifications when from github because of MT releases but currently, there are **NO NEW** releases, the lastest is MT 24.1

**Comment by wolfgang-ch** [comment]
All latest MT releases from 2020-2024 are now available here https://github.com/mytourbook/mytourbook/releases

Download statistics can be displayed with 

* https://tooomm.github.io/github-release-stats/?username=mytourbook&repository=mytourbook
* https://qwertycube.com/github-release-stats/

![image](https://github.com/mytourbook/mytourbook/assets/1283445/8ddc7fae-9e9b-4058-acc6-9dc3c3df4d12)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/913fc4ae-ddb1-45d5-96a4-529fa4653040)

**Comment by wolfgang-ch** [comment]
> This is from the admin site:
> 
> ![image](https://private-user-images.githubusercontent.com/1283445/322871492-6674ebd1-37b0-44f2-8051-8c08642b5f96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1Mjk5MzIsIm5iZiI6MTcxMzUyOTYzMiwicGF0aCI6Ii8xMjgzNDQ1LzMyMjg3MTQ5Mi02Njc0ZWJkMS0zN2IwLTQ0ZjItODA1MS04YzA4NjQyYjVmOTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTlUMTIyNzEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmQwYjdhNzI4MDAwYmVlZGRlMTNjODM0NTVmNTVkNzhkMGJhMTEzY2ZkMzBhOTFlZjI3ZDJkOTIxM2JlY2Y5YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.btDnWChchU1MbAoB9WTDVxn282wBbTBTJ20vMrRLdZ8)

After deleting all MT release files 2020...2023, sourceforge is now happy

![image](https://github.com/mytourbook/mytourbook/assets/1283445/2bb0edca-b8a5-4e1c-9250-728878c24de8)

**Comment by wolfgang-ch** [comment]
Yesterday I received another e-mail to reduce the webspace

![image](https://github.com/user-attachments/assets/dcefdd76-a18c-41dc-b951-77a459d25c3e)

When clicking on the largest file link, then I can see that the limit was redurce from 30 GB to 5 GB

![image](https://github.com/user-attachments/assets/b55697f1-a586-4b01-bda7-5db8830b2799)


So I will reduce the webpace according to the above strategy https://github.com/mytourbook/mytourbook/discussions/1322#discussioncomment-9144750

**Comment by wolfgang-ch** [comment]
Now sourceforge is happy, until the next workspace reduction

![image](https://github.com/user-attachments/assets/740174b0-fc09-4e45-88a1-30105a07a044)
